src/libraries/System.DirectoryServices.Protocols/tests/BerConverterTests.cs

```diff
@@ -42,10 +43,15 @@ public static IEnumerable<object[]> Encode_TestData()
             yield return new object[] { "VVVV", new object[] { null, new byte[][] { new byte[] { 0, 1, 2, 3 }, null }, new byte[][] { new byte[0] }, new byte[0][] }, new byte[] { 4, 4, 0, 1, 2, 3, 4, 0, 4, 0 } };
         }
 
-        [Theory]
+        [ConditionalTheory]
         [MemberData(nameof(Encode_TestData))]
         public void Encode_Objects_ReturnsExpected(string format, object[] values, byte[] expected)
         {
+            if (PlatformDetection.IsArm64Process)
```

Comment:
I'm fine with the change as is but just wondering why you didn't just create a static bool that checks if this is arm64 and pass that as a parameter to the Conditional Theory.

Reply:
Just because this way we don't need to create an extra static property and also, we can customize the message that is printed out to the console logs when showing the skipped reason.

Reply:
at the end it is the same, just the message printed to the console is different.

Reply:
This feature is most to conditionally disable based on the inputs. It seems like it only fails when format is `tetie`. I can update it to check for the format to not loose coverage in the other test data.

Reply:
Updated to only skip for `tetie` format.

Reply:
I would’t worry about that, I plan on fixing this tomorrow and reenable the test so the way you have it is fine.

Reply:
I just did before you commented 🤦